apps/hygeia_web/lib/hygeia_web/live/row_live/apply.ex

```diff
@@ -117,6 +118,28 @@ defmodule HygeiaWeb.RowLive.Apply do
     end)
   end
 
+  def handle_event(
+        "patch_select_case",
+        %{
+          "select_case" => %{
+            "index" => index,
+            "person_uuid" => person_uuid,
+            "case_uuid" => case_uuid,
+            "status_done" => status_done,
+            "suppress_quarantine" => suppress_quarantine
+          }
+        } = _params,
+        socket
+      ) do
+    patch_action_plan(socket, index, fn %Action.SelectCase{} = action ->
+      %Action.SelectCase{
+        action
+        | status_done: status_done,
+          suppress_quarantine: suppress_quarantine
```

Comment:
TODO: convert to boolean

```suggestion
          suppress_quarantine: suppress_quarantine in ["true", "1"]
```